conf/application.conf

```diff
@@ -107,6 +107,7 @@ auditing {
 
 mongodb {
   uri = "mongodb://localhost:27017/transit-movements"
+  retryAttempts = 0
```

Comment:
This is for local testing, if you set anything other than 0, it writes multiple entries into the collection.
In fact, using the test connection string, local retry functionality is disabled (will only work on a proper Mongo cluster).
I have a app-config change to push to set retryAttempts in real environments.

Reply:
Just to quickly check - was this done? If not, we'll just need to push this up then redeploy the various internal environments!

Reply:
@drnaylor-oc working on it, just going to set it to a non-zero value (i.e. 3) in the base config. Don’t think it's worth setting it to different values in different environments.

Reply:
BTW I don’t  think the uri needs to be defined since it is set by MongoComponent which does not seem to be under our control. Seems like the framework sorts that out. So I will only set the `retryAttempts`.